### DISCUSSION TITLE
vc-measurements组件坐标点绘制时替换成广告牌怎么实现？

### DISCUSSION THREAD

**Discussion by 2452615812** [zouyaoji/vue-cesium#451] [Q&A]
vc-measurements组件坐标点绘制时，把点替换成广告牌怎么实现？
![8bcbd42bcef4905043d5eecb582611b](https://github.com/zouyaoji/vue-cesium/assets/92239463/ef37d813-a8e7-4826-af9f-488f730b0d3f)

**Comment by zouyaoji** [comment]
你好，vc-measurements是量算组件。你要绘制广告牌可以考虑用 vc-drawings
